tests/subsys/fs/nffs_fs_api/basic/testcase.yaml

```diff
@@ -3,4 +3,4 @@ common:
 tests:
   filesystem.nffs.basic:
     extra_args: TEST=basic
-    platform_whitelist: qemu_x86
+    platform_whitelist: qemu_x86 nrf52840_pca10056
```

Comment:
Should this also include: nrf51_pca10028 & nrf52_pca10040?

Looking at tests/subsys/fs/nffs_fs_api/basic/CMakeLists.txt I see:

```
if(BOARD STREQUAL nrf51_pca10028)
    set(CONF_FILE nrf5x.conf)
elseif(BOARD STREQUAL nrf52_pca10040)
    set(CONF_FILE nrf5x.conf)
elseif(BOARD STREQUAL nrf52840_pca10056)
    set(CONF_FILE nrf5x.conf)
endif()
```

Reply:
They won't fit in RAM, the test requires 230KB of RAM and the 52840 is the only one with 256KB

Reply:
> They won't fit in RAM, the test requires 230KB of RAM and the 52840 is the only one with 256KB

So should those be removed from the CMakeLists?

Reply:
I have now